views/default/elgg.js.php

```diff
@@ -98,3 +98,5 @@
 }
 
 elgg.trigger_hook('boot', 'system');
+
+require(['elgg/init', 'elgg/ready']);
```

Comment:
We don't need to load these modules on each request. Also I think it's safer if these are here, so plugins extending this view don't delay these events.